mods/ts/rules/shared-support.yaml

```diff
@@ -3,8 +3,8 @@ NAPULS:
 	Valued:
 		Cost: 1000
 	Tooltip:
-		Name: EMP Cannon
-		Description: Disables vehicles.\nRequires power to operate.\n  Strong vs Vehicles\n  Weak vs Infantry, Aircraft
+		Name: E.M. Pulse Cannon
+		Description: Disables vehicles in an area.\nRequires power to operate.
```

Comment:
It says `mechanical units` elsewhere and we also disable cyborgs: https://github.com/OpenRA/OpenRA/blob/bleed/mods/ts/rules/defaults.yaml#L370